frontend/src/v5/ui/controls/inputs/textArea/textAreaFixedSize.component.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { FormControl, FormHelperText, InputLabel, InputProps } from '@mui/material';
+import { FormInputProps } from '@controls/inputs/controlledInput.component';
+import { ScrollArea } from '@controls/scrollArea';
+import { Container, Input } from './textAreaFixedSize.styles';
+
+export type TextAreaFixedSizeProps = FormInputProps & InputProps & {
+	height?: number,
+};
+
+export const TextAreaFixedSize = ({
+	error,
+	helperText,
+	name,
+	required,
+	disabled,
+	label,
+	value = '', // this is to be certain that is a controlled field
+	height = 80,
+	...props
+}: TextAreaFixedSizeProps) => (
+	<FormControl required={required} disabled={disabled} error={error}>
+		{label && (
+			<InputLabel id={`${name}-label`}>
+				{label}
+			</InputLabel>
+		)}
```

Comment:
This problem only arises in Firefox as React-custom-scrollbars hides away and Firefox uses its own scrollbar. This branch is not really concerned with styling, so I guess it's better to create a new issue in case to address this problem

Reply:
Added the bug to this list: https://github.com/3drepo/3drepo.io/issues/3808